backend/db/db.js

```diff
@@ -0,0 +1,5 @@
+import { PrismaClient } from '@prisma/client'
+
+const prisma = new PrismaClient()
+
+module.exports = prisma
```

Comment:
to stay consistent with our module system, lets use `export` instead of `module.exports` A quick fix would be to change to this

```js 
export default prisma;
```